app/models/history.rb

```diff
@@ -113,6 +114,31 @@ class History < ApplicationRecord
       .where(history_type: [HISTORY_TYPES[:reports_reviewed], HISTORY_TYPES[:report_reviewed]])
   }
 
+  def self.errored_report_reminder_group_of_patients(patients: nil, created_by: 'Sara Alert System', comment: 'Failed Contact Attempt', error_message: nil)
+    histories = []
+    patients.uniq.each do |pat|
+      if pat.responder == pat
+        recipient = 'this monitoree'
+        responder = pat
+      else
+        recipient = "this monitoree's head of household"
+        responder = pat.responder
+      end
+      details = if !error_message.nil?
```

Comment:
This is more of a style thing but this could be one line instead of three:
```details = error_message.present? ? ' Error details: ' + error_message : ''```